app/routes/cph-list.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  method: 'GET',
+  path: '/cph-list',
+  options: {
+    auth: false,
```

Comment:
Worth removing this? I just did it locally to check the auth strategy was working

Reply:
I'll not do it as part of this PR as it has been changed in #6